lib/helper/WebDriver.js

```diff
@@ -244,11 +244,12 @@ class WebDriver extends Helper {
   _validateConfig(config) {
     const defaults = {
       // webdriverio defaults
-      protocol: 'http',
-      host: '0.0.0.0',
-      port: 4444,
-      path: '/wd/hub',
-      logLevel: 'error',
```

Comment:
Log level `error` must be set to disable verbose output.
All other defaults can be removed... I think

Reply:
updated code as your suggestion